pkgs/by-name/he/headplane/package.nix

```
@@ -0,0 +1,51 @@
+{
+  fetchFromGitHub,
+  git,
+  lib,
+  makeWrapper,
+  nodejs_22,
+  pnpm_10,
+  stdenv,
+  ...
+}:
+stdenv.mkDerivation (finalAttrs: rec {
```

rec and finalAttrs server the same purpose  
```suggestion
stdenv.mkDerivation (finalAttrs: {
```